tests/Agent/IntegrationTests/UnboundedIntegrationTests/Oracle/OracleStoredProcedureTests.cs

```diff
@@ -60,48 +63,85 @@ public void Test()
         {
             var expectedMetrics = new List<Assertions.ExpectedMetric>
             {
-                new Assertions.ExpectedMetric { metricName = $@"Datastore/statement/Oracle/{_procedureName.ToLower()}/ExecuteProcedure", callCount = 1 },
-                new Assertions.ExpectedMetric { metricName = $@"Datastore/statement/Oracle/{_procedureName.ToLower()}/ExecuteProcedure", callCount = 1, metricScope = "WebTransaction/MVC/OracleController/OracleParameterizedStoredProcedure"}
+                new() { metricName = $@"Datastore/statement/Oracle/{_storedProcedureName.ToLower()}/ExecuteProcedure", callCount = 1 },
```

Comment:
Is there an advantage to the `new()` style as opposed to being explicit about what kind of Assertions. object we're newing up?

Reply:
It's similar to `var` in that the compiler chooses the natural type. In this case it will be `Assertions.ExpectedMetric` because of its use in the List initializer. If you needed it to be some other type, or a child type, then you would need to use the explicit type constructor.

Reply:
I prefer that style as it's a whole lot less unnecessary typing on my part (and a lot less complaining from ReSharper... :wink:)